pkg/options/options.go

```diff
@@ -101,6 +101,22 @@ var MongoDbURL = ""
 // it can be overridden for testing or different url.
 var PostgresDbURL = ""
 
+// PostgresSSLRequired is used for forcing the client to work with SSL
+// when using an external postgres DB.
+var PostgresSSLRequired = false
+
+// PostgresSSLSelfSigned is used for forcing the client to work with self-signed SSL
+// when using an external postgres DB.
+var PostgresSSLSelfSigned = false
+
+// PostgresSSLKey is used for providing postgres url
+// it can be overridden for testing or different url.
+var PostgresSSLKey = ""
+
+// PostgresSSLCert is used for providing postgres url
+// it can be overridden for testing or different url.
+var PostgresSSLCert = ""
```

Comment:
```suggestion
// PostgresSSLRequired is used to force the client to work with SSL
// when using an external postgres DB.
var PostgresSSLRequired = false

// PostgresSSLSelfSigned is used to force the client to work with self-signed SSL
// when using an external postgres DB.
var PostgresSSLSelfSigned = false

// PostgresSSLKey is used for providing postgres url
// it can be overridden for testing or different url.
var PostgresSSLKey = ""

// PostgresSSLCert is used for providing postgres url
// it can be overridden for testing or a different url.
var PostgresSSLCert = ""
```

Reply:
also, most of it is incorrect. thanks, fixed